modules/vmseries/main.tf

```diff
@@ -3,13 +3,22 @@ resource "azurerm_public_ip" "this" {
 
   location            = var.location
   resource_group_name = var.resource_group_name
-  name                = each.value.name
+  name                = "${each.value.name}-pip"
```

Comment:
I can see this suffix being added throughout the PR, in different modules. I think we should avoid any static suffixes in resource names whenever possible.

Reply:
I do not see a place where they could harm the code or usability. Especially that these are only names, we operate on keys everywhere, so the code does not depend on them. 

And it is more readable when you look in Azure portal. Keeping the naming schema, where each related resource get's a prefix in the form of the main resource + some suffix that usually describes a type of a resource groups the related resources together: 
![image](https://user-images.githubusercontent.com/42772730/234247998-5ffdb681-70f4-4d12-90a9-60996e2c8054.png)

Reply:
what we could do is work on keeping the suffixes the same across modules. Or even make them configurable, example: public IP get's `-pip` but you can overwrite it with a variable (one variable for all PIP suffixes)